webview-ui/src/components/settings/providers/LMStudioProvider.tsx

```diff
@@ -151,6 +157,9 @@ export const LMStudioProvider = ({ currentMode }: LMStudioProviderProps) => {
 				value={String(currentLoadedContext ?? lmStudioMaxTokens ?? "-")}
 			/>
 
+			<VSCodeCheckbox checked={isCompactPromptEnabled} onChange={() => toggleCompactPrompt(!isCompactPromptEnabled)}>
+				Use Compact Prompt
```

Comment:
Any helper text we want to add here?

Reply:
<img width="432" height="154" alt="image" src="https://github.com/user-attachments/assets/bb7ab4fd-7910-45a3-93d8-92f633e90ac8" />

Reply:
updated error message in lmstuiod.ts and description text under the checkbox